### PR TITLE
🍱 protondrive added

### PR DESCRIPTION
**Pull request by haykh**
Solves #50

### CODE REVIEW

File: recipes/protondrive/webview.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+module.exports = (Ferdium) => {
```
Comment (vraravam) [inline recipes/protondrive/webview.js:5]: If the config setting in `package.json` has `hasIndirectMessages` as `true`, then there should be some corresponding code in this module to call `Ferdium.setBadge()`. Without such a call, the badges won't appear. Is that ok?

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by haykh** [comment]
Should be fine. I'm actually not sure if they ever send any browser notifications, never seen one. If this changes -- I'll update it accordingly.

File: recipes/protondrive/webview.js (hunk 1 above)
Comment (haykh) [reply recipes/protondrive/webview.js:5]: Should be fine. I'm actually not sure if they ever send any browser notifications, never seen one. If this changes -- I'll update it accordingly.

**Review by vraravam — CHANGES_REQUESTED** [review]

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@haykh - please run `pnpm package` to check and fix the above issue.

**Comment by vraravam** [comment]
@all-contributors please add @haykh for code

**Pull request merged**